scripts/cangen/main.py

```diff
@@ -45,6 +48,16 @@ def parse():
     args = parse()
     project_folder_name = args.project
     os.chdir(os.path.join(DIR_PROJECTS, project_folder_name))
+    
+    # Map CMAKE log levels -> Python log levels
+    log_level_mappings = {
+        "STATUS": "INFO",
+        "INFO": "INFO",
+        "VERBOSE": "DEBUG",
+        "DEBUG": "DEBUG"
+    }
+    # Set log level thrershold to specified verbosity
+    logging.getLogger().setLevel(log_level_mappings[args.level])
```

Comment:
This line will cause a KeyError if a less verbose cmake level is used (like ERROR or WARNING) since they are not keys in the dictionary. Use `log_level_mappings.get(args.level, "INFO")` which will default to INFO if the key is not found.

Reply:
Nevermind, i see that this case is avoided by the argparse argument choices.